examples/kit-nextjs-article-starter/.sitecore/component-map.client.ts

```diff
@@ -0,0 +1,89 @@
+import * as VerticalImageAccordion from 'src/components/vertical-image-accordion/VerticalImageAccordion';
+import * as TopicItemdev from 'src/components/topic-listing/TopicItem.dev';
+import * as themeproviderdev from 'src/components/theme-provider/theme-provider.dev';
+import * as TestimonialCarousel from 'src/components/testimonial-carousel/TestimonialCarousel';
+import * as Title from 'src/components/sxa/Title';
+import * as PageContent from 'src/components/sxa/PageContent';
+import * as Navigation from 'src/components/sxa/Navigation';
+import * as Image from 'src/components/sxa/Image';
+import * as SubscriptionBanner from 'src/components/subscription-banner/SubscriptionBanner';
+import * as SecondaryNavigation from 'src/components/secondary-navigation/SecondaryNavigation';
+import * as PromoAnimatedImageRightdev from 'src/components/promo-animated/PromoAnimatedImageRight.dev';
+import * as PromoAnimatedDefaultdev from 'src/components/promo-animated/PromoAnimatedDefault.dev';
+import * as PromoAnimated from 'src/components/promo-animated/PromoAnimated';
+import * as portaldev from 'src/components/portal/portal.dev';
+import * as PageHeader from 'src/components/page-header/PageHeader';
+import * as MultiPromoTabs from 'src/components/multi-promo-tabs/MultiPromoTabs';
+import * as MultiPromo from 'src/components/multi-promo/MultiPromo';
+import * as modetoggledev from 'src/components/mode-toggle/mode-toggle.dev';
+import * as MediaSectiondev from 'src/components/media-section/MediaSection.dev';
+import * as meteors from 'src/components/magicui/meteors';
+import * as LogoTabs from 'src/components/logo-tabs/LogoTabs';
+import * as nextImageSrcdev from 'src/components/image/nextImageSrc.dev';
+import * as ImageWrapperdev from 'src/components/image/ImageWrapper.dev';
+import * as imageoptimizationcontext from 'src/components/image/image-optimization.context';
+import * as Icon from 'src/components/icon/Icon';
+import * as Hero from 'src/components/hero/Hero';
+import * as GlobalHeader from 'src/components/global-header/GlobalHeader';
+import * as FooterNavigationColumn from 'src/components/global-footer/FooterNavigationColumn';
+import * as floatingdockdev from 'src/components/floating-dock/floating-dock.dev';
+import * as CtaBanner from 'src/components/cta-banner/CtaBanner';
+import * as BackgroundThumbnaildev from 'src/components/background-thumbnail/BackgroundThumbnail.dev';
+import * as ArticleListing from 'src/components/article-listing/ArticleListing';
+import * as ArticleHeader from 'src/components/article-header/ArticleHeader';
+import * as AlertBannerdev from 'src/components/alert-banner/AlertBanner.dev';
+import * as AccordionBlock from 'src/components/accordion-block/AccordionBlock';
+
+export const componentMap = new Map<string, NextjsContentSdkComponent>([
+  ['BYOCWrapper', BYOCClientWrapper],
+  ['FEaaSWrapper', FEaaSClientWrapper],
+  ['Form', Form],
+  ['VideoPlayer', { ...VideoPlayerdev }],
+  ['VideoModal', { ...VideoModaldev }],
```

Comment:
some components, like this one, are present in both client and default component maps. Please ensure they are present in only one map.
one more thing: if `VideoModal` is a variant of a `Video` rendering/component, please register it as a such in component map:
 https://doc.sitecore.com/sai/en/developers/content-sdk/register-a-component-in-the-component-map.html#configure-a-dynamic-import-to-use-one-or-more-variants